tests/manage/pv_services/test_run_io_multiple_pods.py

```diff
@@ -1,103 +1,54 @@
 import pytest
 import logging
-import random
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
 from ocs_ci.ocs import constants
 
 from ocs_ci.framework.testlib import ManageTest, tier2
-from tests.fixtures import (
-    create_rbd_storageclass, create_ceph_block_pool,
-    create_rbd_secret, create_pods, create_pvcs, create_project,
-    create_cephfs_secret, create_cephfs_storageclass
-)
 
 
 logger = logging.getLogger(__name__)
```

Comment:
`logger` is not being used later in the script, I think can be removed

Reply:
right, removed